### ISSUE TITLE
Bold text style matches everything?

### ISSUE THREAD

**Issue by heigren** [hSaria/ChromaTerm#73]
## Description
When using the "bold" it matches everything after the initial match, while the color configured on the same rule is displaying correctly. The bold style continues until the session is closed. `ct --reload` does not help either.

![Screenshot from 2019-10-09 11-33-33](https://user-images.githubusercontent.com/726720/66469765-bdaa0980-ea88-11e9-880d-57033d321f42.png)

## Environment
-   OS: Ubuntu 19.04
-   ChromaTerm: 0.4.8

## Configuration
```yaml
  regex: (?:[0-9]*[1-9][0-9]*\W|^)((input errors|errors|error|err|reset|act\/unsup|dhcp|DHCP|mismatch|notconnect|drops|dropped|runts|CRC|collisions|collision|LRN|learning
|listening|LIS|unsynchronized))(?=[\W\s]|$)
  color: f#c94b4b bold
```

**Comment by hSaria** [comment]
Thanks for the submission. I was able to recreate the problem on v0.5.0. I'll get a fix out ASAP.

On initial inspection, it seems like ct is doing what it's designed to do:
```
v color start                        v color reset
\e[38;5;167m\e[1m1 input errors\e[21m\e[39m
            ^ bold on          ^ bold off
```
I looked at a [Wiki page](https://en.wikipedia.org/wiki/ANSI_escape_code) for getting the "bold off" (`\e[21m`) code but it would seems the it is not [widely supported](https://en.wikipedia.org/wiki/Talk:ANSI_escape_code#SGR_21—`Bold_off`_not_widely_supported). I'll find the correct code or an alternative for bold off.

**Comment by hSaria** [comment]
Aha! Found it! It's `\e[22m` for "normal intensity".

**Comment by hSaria** [comment]
Fixed in 7b86041b85458073b970dc17fce43a6ae4846b7a. v0.5.1 published. Thanks again for creating this issue.
